wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -480,6 +485,45 @@ instance ToSchema NewWallet where
       & ("operation"        --^ "Create a new wallet or Restore an existing one.")
     )
 
+-- | A type modelling the request for a new 'ExternalWallet',
+-- on the mobile client or hardware wallet.
+data NewExternalWallet = NewExternalWallet
+    { newewalExtPubKey      :: !Text            -- ^ Base58-encoded extended public key.
```

Comment:
I wish we had a type that could express this well. A `newtype` around `ByteString` would be much safer than an arbitrary Text encoding.